docs/UserGuide.md

```diff
@@ -1071,8 +1071,8 @@ Format: `list-r`
 Fig 11. Revenue bar chart
 {:.caption}
 
-#### 4.7.2 Viewing top 10 clients
-* Your top 10 clients who made the highest number of orders will be shown in a pie chart.
+#### 4.7.2 Viewing top 10 most frequent clients
+* Your top 10 most frequent clients, who made the highest number of orders will be shown in a pie chart.
```

Comment:
Could you add a line explaining that this counts the number of orders, regardless of order quantity?